examples/examples_test.go

```diff
@@ -86,83 +86,86 @@ func TestExamples(t *testing.T) {
+			// 		assert.Equal(t, tokens.Type("kubernetes:apps/v1:Deployment"), frontendDepl.URN.Type())
+			// 		name, _ = openapi.Pluck(frontendDepl.Outputs, "live", "metadata", "name")
+			// 		assert.Equal(t, "frontend", name)
+			// 		status, _ = openapi.Pluck(frontendDepl.Outputs, "live", "status", "readyReplicas")
+			// 		assert.Equal(t, float64(3), status)
+
+			// 		// Verify redis-master deployment.
+			// 		redisMasterDepl := stackInfo.Deployment.Resources[1]
+			// 		assert.Equal(t, tokens.Type("kubernetes:apps/v1:Deployment"), redisMasterDepl.URN.Type())
+			// 		name, _ = openapi.Pluck(redisMasterDepl.Outputs, "live", "metadata", "name")
+			// 		assert.Equal(t, "redis-master", name)
+			// 		status, _ = openapi.Pluck(redisMasterDepl.Outputs, "live", "status", "readyReplicas")
+			// 		assert.Equal(t, float64(1), status)
+
+			// 		// Verify redis-slave deployment.
+			// 		redisSlaveDepl := stackInfo.Deployment.Resources[2]
+			// 		assert.Equal(t, tokens.Type("kubernetes:apps/v1:Deployment"), redisSlaveDepl.URN.Type())
+			// 		name, _ = openapi.Pluck(redisSlaveDepl.Outputs, "live", "metadata", "name")
+			// 		assert.Equal(t, "redis-slave", name)
+			// 		status, _ = openapi.Pluck(redisSlaveDepl.Outputs, "live", "status", "readyReplicas")
+			// 		assert.Equal(t, float64(1), status)
+
+			// 		// Verify frontend service.
+			// 		frontentService := stackInfo.Deployment.Resources[3]
+			// 		assert.Equal(t, tokens.Type("kubernetes:core/v1:Service"), frontentService.URN.Type())
+			// 		name, _ = openapi.Pluck(frontentService.Outputs, "live", "metadata", "name")
+			// 		assert.Equal(t, "frontend", name)
+			// 		status, _ = openapi.Pluck(frontentService.Outputs, "live", "spec", "clusterIP")
+			// 		assert.True(t, len(status.(string)) > 1)
+
+			// 		// Verify redis-master service.
+			// 		redisMasterService := stackInfo.Deployment.Resources[4]
+			// 		assert.Equal(t, tokens.Type("kubernetes:core/v1:Service"), redisMasterService.URN.Type())
+			// 		name, _ = openapi.Pluck(redisMasterService.Outputs, "live", "metadata", "name")
+			// 		assert.Equal(t, "redis-master", name)
+			// 		status, _ = openapi.Pluck(redisMasterService.Outputs, "live", "spec", "clusterIP")
+			// 		assert.True(t, len(status.(string)) > 1)
+
+			// 		// Verify redis-slave service.
+			// 		redisSlaveService := stackInfo.Deployment.Resources[5]
+			// 		assert.Equal(t, tokens.Type("kubernetes:core/v1:Service"), redisSlaveService.URN.Type())
+			// 		name, _ = openapi.Pluck(redisSlaveService.Outputs, "live", "metadata", "name")
+			// 		assert.Equal(t, "redis-slave", name)
+			// 		status, _ = openapi.Pluck(redisSlaveService.Outputs, "live", "spec", "clusterIP")
+			// 		assert.True(t, len(status.(string)) > 1)
+
+			// 		// Verify root resource.
+			// 		stackRes := stackInfo.Deployment.Resources[6]
+			// 		assert.Equal(t, resource.RootStackType, stackRes.URN.Type())
+			// 	},
+			// }),
+
+			// TODO(hausdorff): Enable this when parallelism is turned on.
```

Comment:
In general we've been aiming to track TODO with issues and link them in code.  That might be nice to do here and link t existing parallelism issues.